lib/cretonne/meta/isa/riscv/registers.py

```diff
@@ -18,6 +18,8 @@
         units=32, prefix='f')
 
 GPR = RegClass(IntRegs)
+# The popular registers used in some compressed instructions; x8-x15
+GPRC = GPR[8:15]
```

Comment:
This is using Python's slice semantics, so you want `8:16`.

Reply:
Ah of course! I'm a bit rusty on Python